WooCommerce.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -84,7 +84,7 @@
       },
       {
         "package": "swift-numerics",
-        "repositoryURL": "https://github.com/apple/swift-numerics.git",
```

Comment:
This change has been in and out for a while, I'm wondering why 🤔

Reply:
Ah, that snuck in here. I've usually been discarding that change when it pops up.

I've seen other people wondering about this and haven't found a solid answer, but I suspect the variation is because the dependency URL is declared as `https://github.com/apple/swift-numerics` instead of `https://github.com/apple/swift-numerics.git` and something, somewhere, likes to make the `.git` extension explicit (so it gets added back in from time to time). Since this is a sub-dependency I'm not sure there's much we can do about it unless we want to remove this file from version control.